extensions/amp-minute-media-player/0.1/amp-minute-media-player.js

```diff
@@ -198,11 +198,10 @@ class AmpMinuteMediaPlayer extends AMP.BaseElement {
    * @private
    */
   iframeSource_() {
-    const baseUrl =
-      'https://www.oo-syringe.com/dev/amp/mplayer.html' +
-      `?content_type=${encodeURIComponent(this.contentType_)}`;
+    const baseUrl = 'https://www.oo-syringe.com/dev/amp/mplayer.html';
 
     const moreQueryParams = dict({
+      'content_type': this.contentType_ || undefined,
```

Comment:
Wouldn't these values be undefined either way?

Reply:
No, these value would be an empty strings or null and it would be part of the url for no reason.

Reply:
We have an error (in the remote test) that we didn't have before (as you can see commit #42ccc3456ff3283c515a304e8f07905caa0d123e) and I believe that this pr was not the cause of this error. Can you please advise?
Furthermore, I would like to know if there is something else we should change\add and if not, how can we proceed with the process.
Thank you!

Reply:
Hi  Alan, any news?

Reply:
Hi,
I rebase with master and all the checks have passed.
Would very appreciate your input on this.
Thanks

Reply:
LGTM! Thanks!

Reply:
Thank you! Very happy to get your approval (:
How can we proceed towards merging our component into master?
What are the two checks that are neutral?